src/store/portal/index.js

```diff
@@ -76,6 +79,9 @@ module.exports = {
       }
 
       return vfetch(opts)
+              .then((res) => {
+                dispatch('getHostGroupList')
+              })
```

Comment:
fixed in https://github.com/cepave-f2e/owl-light/pull/59/commits/8059f56ac361ceb58cda16739807d24798003be9